core/src/main/java/edu/wpi/grip/core/Operation.java

```diff
@@ -61,7 +61,6 @@ default void perform(InputSocket<?>[] inputs, OutputSocket<?>[] outputs, Optiona
         perform(inputs, outputs);
     }
 
-    @Deprecated
```

Comment:
I think this method should actually be removed at some point. Why do you disagree?
Also, why is this part of this PR?